image/preseed/reset.go

```diff
@@ -31,6 +31,50 @@ import (
 	apparmor_sandbox "github.com/snapcore/snapd/sandbox/apparmor"
 )
 
+// bash-completion symlinks; note there are symlinks that point at
+// completer, and symlinks that point at the completer symlinks.
+// e.g.
+// lxd.lxc -> /snap/core/current/usr/lib/snapd/complete.sh
+// lxc -> lxd.lxc
+func resetCompletionSymlinks(completersPath string, preseedChroot string) error {
+	files, err := ioutil.ReadDir(filepath.Join(preseedChroot, completersPath))
+	if err != nil && !os.IsNotExist(err) {
+		return fmt.Errorf("error reading %s: %v", completersPath, err)
```

Comment:
here it might make sense to print the path with the preseedChroot prefix as well

Reply:
Done